### PR TITLE
Clarify FS GCP capabilities

### PR DESCRIPTION
**Pull request by azure562**

### CODE REVIEW

**Review by yossi77 — APPROVED** [review]
Thank you

**Review by norosa — DISMISSED** [review]

File: website/docs/flexsave/gcp.mdx [hunk 1]
```diff
@@ -19,9 +19,18 @@ In most cases, save [the equivalent of a 1-year commitment discount](overview.md
...
 ## Capabilities
 
-### Supported regions and machine types
+### Supported commitment types
 
-Flexsave supports all GCP [regions](https://cloud.google.com/compute/docs/regions-zones) and [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs (except for GPU accelerators, i.e., `A2` instances). We don't support [commitments for GPUs or SSDs](https://cloud.google.com/compute/docs/instances/signing-up-committed-use-discounts#commitments_for_gpus_and_local_ssd).
+Flexsave supports:
+
+- all GCP [regions](https://cloud.google.com/compute/docs/regions-zones)
+- most of the [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs
```
[the remark this answers is not in the document]
Comment (azure562) [reply website/docs/flexsave/gcp.mdx]: the not supported types are in the following section

File: website/docs/flexsave/gcp.mdx [hunk 2]
```diff
@@ -19,9 +19,18 @@ In most cases, save [the equivalent of a 1-year commitment discount](overview.md
...
 ## Capabilities
 
-### Supported regions and machine types
+### Supported commitment types
 
-Flexsave supports all GCP [regions](https://cloud.google.com/compute/docs/regions-zones) and [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs (except for GPU accelerators, i.e., `A2` instances). We don't support [commitments for GPUs or SSDs](https://cloud.google.com/compute/docs/instances/signing-up-committed-use-discounts#commitments_for_gpus_and_local_ssd).
+Flexsave supports:
+
+- All GCP [regions](https://cloud.google.com/compute/docs/regions-zones)
+- Most of the [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs
```
Comment (norosa) [inline website/docs/flexsave/gcp.mdx:27]:
```suggestion
- Most [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs (see below for exceptions)
```

What about something like this? I still feel that we should qualify the "most".

Comment (azure562) [reply website/docs/flexsave/gcp.mdx:27]: as they are immediately adjacent, I think it's fine

**Comment by norosa** [comment]
@azure562 looks good to me. Approved. Merge away!

P.S., I added the ready-to-merge label, which should signal to Mergify to automatically merge the PR. However, it looks like the configuration is messed up somehow. Maybe a temporary thing.

**Pull request merged**